hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/overview/overview.tsx

```diff
@@ -302,6 +317,11 @@ export class Overview extends React.Component<Record<string, object>, IOverviewS
           <Col xs={24} sm={18} md={12} lg={12} xl={6} className='summary-font'>
             <OverviewCard loading={loading} title='Pending Deleted Keys Summary' data={deletePendingSummaryData} icon='delete' linkToUrl='/Om'/>
           </Col>
+          {clusterIsInHaMode &&
+            <Col xs={24} sm={18} md={12} lg={12} xl={6}>
+              <OverviewCard loading={loading} data={scmAndOmServicesData}/>
```

Comment:
Could you help to take a screenshot on the overview page and update the PR description?

You can either use `pnpm run dev` with mocked data, or you spin up a docker compose cluster.

We might want to consider adding an icon and also having one `OverviewCard` for each service ID (you can set to OM service ID overview card to link to '/Om' page).